api/control.proto

```diff
@@ -29,6 +41,9 @@ service Control {
 	rpc ListTasks(ListTasksRequest) returns (ListTasksResponse) {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-manager" };
 	};
+	rpc Attach(stream TaskExecStream) returns (stream TaskExecStream) {
+		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-manager"};
+	}
```

Comment:
If this PR moves forward, this should be part of a separate service instead of adding to controlapi. You can look at logs as an example. Similarly, `TaskExecStream` and `TaskExec` above should move out of this file.

Reply:
Thanks for pointing me to how the logsbroker is impelemented. Probably the whole "stdout" part can be taken from here.